clientv3/balancer.go

```diff
@@ -162,6 +163,18 @@ func (b *simpleBalancer) Up(addr grpc.Address) func(error) {
 
 func (b *simpleBalancer) Get(ctx context.Context, opts grpc.BalancerGetOptions) (grpc.Address, func(), error) {
 	var addr string
+
+	if !opts.BlockingWait {
```

Comment:
doc string this?

```
If opts.BlockingWait is false (for fail-fast RPCs), it should return an address it has notified via Notify immediately instead of blocking.
```